uiowa/uiowa
# Hotfix: Remove required property from mentor fields and other updates

# Updates
* Removes required property from mentor-related fields.
* Updates field machine names for some mentor-related fields.
* Resolves #2620: Adds a college field to mentor person type, currently scoped to grad site split.
* Updates entity displays based on changes above.
* Updates mentor person type with field changes.
* Removed department field to prepare for #2622.

# How to test

* `blt ds --site=grad.uiowa.edu`
* `drush @grad.local uli node/71/edit`
* Save node without making changes.
* Observe no form errors.
* Review [files](https://github.com/uiowa/uiowa/pull/2684/files) to make sure that all changes are within the grad.uiowa.edu site split.
